osh.spec

```diff
@@ -233,6 +233,7 @@ fi
 
 %files hub
 %defattr(-,root,apache,-)
+%{_sbindir}/osh-resalloc
```

Comment:
Is `/sbin` the correct path to store this script?

Reply:
`%{_sbindir}` expands to `/usr/sbin` on a Fedora system.  Assuming root privileges are needed to run the executable, I think the path is correct.  This Fedora page is closely related: https://fedoraproject.org/wiki/Changes/Unify_bin_and_sbin

Reply:
This should be run as exactly the same user and group as the webserver. It is mentioned at the top of the script.

Reply:
Shall this be moved into a separate subpackage?

Reply:
As long as it does nothing in the default configuration and does not introduce any new RPM-level dependencies, I think it is safe to keep it in the existing subpackage.  @lzaoral What is your opinion on this?

Reply:
If we have to find a way to automatically determine number of running nodes, we may end up creating a dependency on the `resalloc` package. In that case, I would move the script to a separate subpackage.